pybandits/quantitative_model.py

```
@@ -605,39 +618,40 @@ def _inner_update(self, segments: List[Segment], rewards: List[BinaryReward], **
             Segments to update.
         rewards : List[BinaryReward]
             Rewards for update.
-        context : Optional[ArrayLike]
-            Context for update.
         """
+        rewards_by_segment = defaultdict(list)
+        for segment, reward in zip(segments, rewards):
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)